src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/CustomProperty.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// MUX Reference CustomProperty.cpp, commit d883cf3
+
+#nullable enable
+
+using System;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.UI.Xaml.Controls;
+
+internal class CustomProperty : ICustomProperty
+{
+	private Func<object, object>? _getter;
+	private Action<object, object>? _setter;
+
+	public CustomProperty(string name, Type type, Func<object, object> getter, Action<object, object> setter)
+	{
+		Name = name;
+		Type = type;
+		_getter = getter;
+		_setter = setter;
+	}
+
+	public bool CanRead => _getter != null;
```

Comment:
I see `_getter` does not change its value after the object is instantiated. Would not be better to assign the value at the constructor?
```suggestion
	public bool CanRead { get; }
```